asyncpg/connection.py

```diff
@@ -112,7 +112,7 @@ def __init__(self, protocol, transport, loop,
         """
         self._check_open()
         if channel not in self._listeners:
-            await self.fetch('LISTEN {}'.format(channel))
+            await self.fetch('LISTEN "{}"'.format(channel))
```

Comment:
Please use `asyncpg.utils._quote_ident()`.

Reply:
Yes, this is what I needed. Thanks.